install.sh

```diff
@@ -659,18 +663,23 @@ do_install() {
 					set -x
 				fi
 				$sh_c "zypper install -y $pre_reqs"
+				$sh_c "rm -f /etc/zypp/repos.d/docker-ce-*.repo"
 				$sh_c "zypper addrepo $repo_file_url"
-				if ! is_dry_run; then
-						cat >&2 <<-'EOF'
-						WARNING!!
-						openSUSE repository (https://download.opensuse.org/repositories/security:/SELinux) will be enabled now.
-						Do you wish to continue?
-						You may press Ctrl+C now to abort this script.
-						EOF
-						( set -x; sleep 30 )
+
+				opensuse_factory_url="https://download.opensuse.org/repositories/security:/SELinux/openSUSE_Factory/"
+				if ! zypper lr -d | grep -q "${opensuse_factory_url}"; then
+					opensuse_repo="${opensuse_factory_url}security:SELinux.repo"
```

Comment:
For this repository, I didn't want to do a blanket "remove the file", because we have no real control over the filename (`/etc/zypp/repos.d/security_SELinux.repo`), which I _think_ is based on `security:SELinux.repo` -> `security_SELinux.repo`, but not 100% sure how much of a promise that is.

So instead of removing the file, I decided to check if a repository pointing to the given URL is already  configured, which should probably cover this situation.